### PR TITLE
initial commit of basic gathering scripts & README

### PR DESCRIPTION
**Pull request by dericcrago**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+### Configuration
+
+Example `.env`
+
+```
+# Required
+CSE_API_KEY='m3DJU4hokX9haNZjvUWKkYnYUTrujAVD3FgWa9e'
+CSE_CX='dezZ22vMRorVt3eVa'
+TWITTER_AUTHORIZATION_HEADER='Bearer WXeXLCEECWy7hxVVXqA4gUybnzAWgj7gMfaFVnZhWvFDkWRU4pUNyb2Ph2s%3kRS7X4ae3gFKX2TfdJ597JxobCdxeQfA3aYURdASpUmw4fTDpf3'
+YOUTUBE_API_KEY='m3DJU4hokX9haNZjvUWKkYnYUTrujAVD3FgWa9e'
```
Comment (GregSutcliffe) [inline README.md]: Should these be public? Or are they made up?

**Comment by GregSutcliffe** [comment]
Seems architecturally fine. We can talk generation of API keys this week and then I can test it

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+[stats]
+bot_name=stats
+bot_version=0.0.1
+bot_author=ansible_community
+
+client_id=5SbCwr35eZNF9Q2DgDdQfA
+client_secret=fbcr3uX5s5uuwwDzwNkuJhnn3VTLDn
+
+username=ansible_community
+password=3RfwMtvWigEqkPvz
```
Comment (GregSutcliffe) [inline README.md]: Likewise here, should these be public? Or are they made up?

File: README.md (hunk 1 above)
Comment (dericcrago) [reply README.md]: these are real-looking examples, but I will update them to be less real-looking

File: README.md (hunk 2 above)
Comment (dericcrago) [reply README.md]: also real-looking examples

File: README.md (hunk 1 above)
Comment (dericcrago) [reply README.md]: removed real-looking examples

File: README.md (hunk 2 above)
Comment (dericcrago) [reply README.md]: removed real-looking examples

**Pull request merged**